docs/verifiable_credentials_examples/talent_protocol_passports/panagiotis_matsinopoulos.json

```diff
@@ -0,0 +1,63 @@
+{
+  "@context": [
+    "https://www.w3.org/ns/credentials/v2",
+    "https://www.w3.org/ns/credentials/examples/v2"
+  ],
+  "id": "https://passport.talentprotocol.com/passports/0x324e9E13dd19528D0F390201923d17c4B7E94462",
+  "type": [
+    "VerifiableCredential",
+    "TalentProtocolPassportCredential"
+  ],
+  "name": "Talent Protocol Credentials",
+  "description": "A verification of Talent Protocol Credentials issued by Talent Protocol to Builders",
+  "issuer": {
+    "id": "https://passport.talentprotocol.com/credentials/issuers/talentprotocol.eth",
+    "name": "Talent Protocol"
+  },
+  "credentialSubject": {
+    "id": "https://etherscan.io/address/0x324e9e13dd19528d0f390201923d17c4b7e94462",
+    "name": "Panagiotis Matsinopoulos", // custom
+    "ens": {
+      "id": "https://passport.talentprotocol.com/credentials/02f9d83f-32b9-4782-83f1-bed7f863c5fc", // passpord_credentials#uuid
+      "name": "PassportCredentials::Ens", // custom
+      "category": "Identity", // custom
+      "validFrom": "2024-04-14T07:12:35Z", // custom taken from onchain_at ? or other datetime
```

Comment:
i feel like we should stick to the validity from our point of view

Reply:
> i feel like we should stick to the validity from our point of view

Yeah. The `validFrom` is an open thing for me. We need to specify how it will be populated. Maybe depending on the claim type to be getting its value from a different source piece of information.